includes/class-wc-order.php

```diff
@@ -1705,6 +1708,18 @@ public function add_order_note( $note, $is_customer_note = 0, $added_by_user = f
 		return $comment_id;
 	}
 
+	/**
+	 * Add an order note for status transition
+	 *
```

Comment:
Please, just include `@since 3.9.0` and everything will be good to merge.

Reply:
Done, thanks.